fabric-biomes-v1/src/testmod/java/net/fabricmc/fabric/test/biome/FabricBiomeTest.java

```diff
@@ -37,5 +43,27 @@
 
 		Biome customPlains = Registry.register(BuiltinRegistries.BIOME, new Identifier(MOD_ID, "custom_plains"), DefaultBiomeCreator.createPlains(null, false));
 		OverworldBiomes.addBiomeVariant(Biomes.PLAINS, customPlains, 1);
+
+		//Loop over existing biomes
+		BuiltinRegistries.BIOME.forEach(this::handleBiome);
```

Comment:
consider using something like <https://github.com/Chainmail-Studios/Astromine/blob/stable-1.16.2/src/main/java/com/github/chainmailstudios/astromine/mixin/DynamicRegistryManagerMixin.java>

this wont work for datapack biomes, also that requires moving the server entrypoint earlier.